src/misc/isEmpty.ts

```diff
@@ -0,0 +1,24 @@
+import { isEmpty as dIsEmpty } from "../date/isEmpty";
+import { isEmpty as sIsEmpty } from "../text/isEmpty";
+import { isEmpty as nIsEmpty } from "../numeric/isEmpty";
+import { isEmpty as bIsEmpty } from "../bool/isEmpty";
+
+/**
+ * @param value
+ * @return boolean
+ */
+export const isEmpty = (value: any): boolean => {
```

Comment:
Instead of using `any`, we could be more specific:

```suggestion
export const isEmpty = (value: boolean | number | string | Date | null | undefined): boolean => {
```